src/snowflake/cli/api/metrics.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2024 Snowflake Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, Optional
+
+_FEATURES_PREFIX = "features"
+_APP_PREFIX = "app"
+
+
+class CLICounterField:
+    TEMPLATES_PROCESSOR = f"{_FEATURES_PREFIX}.templates_processor"
+    SQL_TEMPLATES = f"{_FEATURES_PREFIX}.sql_templates"
+    PDF_TEMPLATES = f"{_FEATURES_PREFIX}.pdf_templates"
+    SNOWPARK_PROCESSOR = f"{_FEATURES_PREFIX}.{_APP_PREFIX}.snowpark_processor"
```

Comment:
I was thinking, automated graph will pick up anything under `features.` Do we want sub commands to also be part of that main graph, or do we want to adjust the hierarchy so that `app` `features` for instance, are tracked in separate graphs? In that case, 'app.features.snowpark_process' would make more sense.

Reply:
@sfc-gh-mchok discussed with Bruno, and I think we can keep features first, but let's add something like 'features.global.' for the global features to keep the structure consistent, and be able to extract global features easily.